### ISSUE TITLE
Overhaul /blog regarding design, content and contributions

### ISSUE THREAD

**Issue by Dovyski** [HaxeFlixel/haxeflixel.com#161]
I think [Haxeflixel's blog](http://haxeflixel.com/blog/) is a great medium to spread tutorials, news and interesting content about the engine. Unfortunately, its design and content organization are quite obsolete. Posts are not related to authors, there is no RSS, no tags, no featured content/post, and so on.

I would like to volunteer myself to work on that front. We don't have to change the blog's design, we just need to make it more friendly regarding content management and contributions. I think it will attract more developers that want to share knowledge. If it becomes easy enough to add content to the blog, maybe the publication frequency will increase.

What are your thoughts on that?

**Comment by Gama11** [comment]
I mean.. it's already quite easy to add blog posts, it's just simple markdown. I feel like the main issue is getting people to actually write blog posts, making the presentation a bit nicer isn't going to help much with that.

Not saying it's a bad idea, improvements are always welcome, but I'm not sure it will necessary have the desired effect.

**Comment by Dovyski** [comment]
I mostly agree with you on this. The main issue is definitely getting people to write blog posts, no questions about that. However, when I had to write something for [/documentation](http://haxeflixel.com/documentation/) it took me a significant amount of work and persistence to see my content rendered locally. Creating a simple markdown file is not the problem, the problem is to iterate and check if the content will be good for the final user _before_ it gets published.

I think this discourage people to contribute more complex content (that are not tweaks). It is simply too much trouble.

> And does anybody still use RSS? ;)

That was an unfair hit, @Gama11 haha! You made me feel very old 😆

**Comment by Gama11** [comment]
>it took me a significant amount of work and persistence to see my content rendered locally.

Hm... Did you run the DocPad build locally for that? If so, that's probably indeed not a good idea... I've basically given up on that and let Travis handle it. :)

Isn't a regular markdown preview good enough for blog posts / docs (for instance the one from GitHub's web editor or VSCode)? That's what I tend to use.

**Comment by Dovyski** [comment]
Yep, I've tried to run DocPad locally. I've almost given up as you said :)

A regular markdown preview should be good enough for the job, but again, the author is limited by the absence of tags, featured content, etc. Everything works now, but there is no "sugar" in the mix, which makes the experience unappealing (for both writers and readers).

**Comment by RichardBray** [comment]
Has a decision been made on this? I think it would be interesting to know the outcome. I don't know how easy it is to add an author and tags feature to the blogs but I also don't know how important that is since there are many other blogging platforms out there that authors use. It'll probably be easier to scrap the flixel blog and create a HaxeFlixel specific medium publication which will have all the features stated here. Here's an example of Netflix's blog - https://netflixtechblog.com/
